### DISCUSSION TITLE
Separate posts that only shows up in relevant page

### DISCUSSION THREAD

**Discussion by Parad0x84** [cotes2020/jekyll-theme-chirpy#1402] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `jekyll-theme-chirpy`

### Description

Basically I'm trying to create a separate kind of post which will not be listed in anywhere including home page, categories page, tag page, etc. It will only show up in a single page (in my case it's portfolio, but it may have other use cases)

I'm still trying to keep benefits of posts like being able to have a toc (table of content), comments, etc and most importantly being able to just write markdown exact same way with posts. (not needing to edit html every time) 

Keep in mind I'm not very familiar with web development and jekyll, but I'm trying my best to understand it.
Also don't think I'm some guy who just asks help without trying or tinkering around. I already did some modifications to the template which I believe at least should demonstrate my dedication

Here is link of the website and repo:
https://paradox84.dev/
https://github.com/Parad0x84/Parad0x84.github.io

I already created an empty page which should list only portfolio posts and an example portfolio post which shouldn't be listed on home page or any other places I mentioned above.

### Operations you have already tried

1-) I did read tutorials, faq, etc inside wiki
2-) I tried to search issues and discussions. Only similar one is this: https://github.com/cotes2020/jekyll-theme-chirpy/discussions/1230#discussion-5617405
3-) I tried to search web for chirpy specific solutions and general solutions for jekyll but didn't find much
4-) I did play around with code, tried different strategies like using categories to split posts and editing stuff inside config file (like adding new defaults). Nothing worked

When I tried using categories or adding new defaults none of it actually filtered the posts and some of them broke the existing listing of posts so invalid posts appeared on home page with no title, content, link and an invalid date (mm, dd, 1970)

**Comment by jakre1234** [comment]
Seems like you achieved this, how did you do that ?
